shard/shard_state.go

```diff
@@ -37,8 +37,14 @@ type State struct {
 	Shards []Committee `json:"shards"`
 }
 
-// BLSPublicKey defines the bls public key
-// TODO(audit): wrap c bls key object with the raw bytes
+// BLSPublicKeyWrapper defines the bls public key in both serialized and
+// deserialized form.
+type BLSPublicKeyWrapper struct {
```

Comment:
The wrapper should probably be in a common package or bls package. Don't think it belongs to `shard` package.

Reply:
Agree. Will take care of it in next PR.